packages/validator/src/validator.ts

```diff
@@ -159,6 +162,32 @@ export class Validator {
       this.clock.start(this.controller.signal);
       this.chainHeaderTracker.start(this.controller.signal);
 
+      const validatorKeys = this.validatorStore.votingPubkeys();
+
+      for (const pubkeyHex in validatorKeys) {
+        let feeRecipient = defaultOptions.suggestedFeeRecipient;
+
+        try {
+          feeRecipient = validatorStore.getFeeRecipient(pubkeyHex);
+        } catch (e) {
+          this.logger.warn("Cannot retrieve fee recipient", {}, e as Error);
+        }
+
+        if (!indicesService.validatorPubKeyExists(pubkeyHex)) {
+          this.logger.info("Validator exists in beacon chain", {
+            index: validatorStore.getIndexOfPubkey(pubkeyHex),
+            pubKey: prettyBytes(pubkeyHex),
+            feeRecipient,
+          });
+        }
+      }
+
+      this.clock.runEveryEpoch(() =>
+        this.fetchValidatorStatuses(validatorKeys, metrics).catch((e) =>
```

Comment:
just a note on this, currently the `Validator statuses` log does not work as expected, we call `fetchValidatorIndices` in batched of 10

https://github.com/ChainSafe/lodestar/blob/887064d7f497adf123a3839ec4c71c323c24bbdd/packages/validator/src/services/indices.ts#L113-L114

The produced log looks like this at the moment which is not correct

```
debug: HttpClient request routeId=getStateValidators
Apr-05 09:48:01.056[]                debug: HttpClient response routeId=getStateValidators
Apr-05 09:48:01.056[]                 info: Validator exists in beacon chain validatorIndex=461922, pubKey=0xa00421a5fad45881c14e2fa23f43aa93c21fd6610f96cc1e9be57c33178648730bdea843d4150d41cc610b857f4a2a59
Apr-05 09:48:01.057[]                 info: Validator exists in beacon chain validatorIndex=461923, pubKey=0xaed788793b98d6bed220cfa06ab357dc9c2bd1512f42f894c76deade02c404826f01758f0d90b6f5544d66a2788a27fe
Apr-05 09:48:01.057[]                 info: Validator exists in beacon chain validatorIndex=461924, pubKey=0xaa052e0d9603559793fea8515e0c6b6ff0e2410d0da08413ff84f1fe371b42f3e3381eb0e7506d423a870de58dc1b9fd
Apr-05 09:48:01.057[]                 info: Validator exists in beacon chain validatorIndex=461925, pubKey=0x8256e393a0332d0bc59f63b4490b3ed838240e6f32ac2409b95cced191542fb944e3866ae2d05d15e5122339c7281677
Apr-05 09:48:01.057[]                 info: Validator exists in beacon chain validatorIndex=461926, pubKey=0x8bfc76d96f53aa973acae88fd6b56a0c2b595e5ed3da2fb6b8353ace436fa0b0278b3ab0ee2c74ee2c93c03729f03958
Apr-05 09:48:01.057[]                 info: Validator exists in beacon chain validatorIndex=461927, pubKey=0xade304c8dda06bcf3cd70906e0f4f278577d530b997ed00159ce4c2dec4ea6586ddabfdbd2b470b5f16a1fdb0bf67c83
Apr-05 09:48:01.057[]                 info: Validator exists in beacon chain validatorIndex=461928, pubKey=0xb72e9beb76cbd5d37fdebff9cd65a2e019816f10b827decf36e149f8c32859c2837feb6edc066dda832253bd1a2d6624
Apr-05 09:48:01.057[]                 info: Validator exists in beacon chain validatorIndex=461929, pubKey=0xb52535102a3ea2d128bab7383c60c2c3e92e5153bbb80aa6b57ffe34051b6a82d5d960833369024f8c5e0392f3fb58fa
Apr-05 09:48:01.057[]                 info: Validator exists in beacon chain validatorIndex=461930, pubKey=0xa1107ab5b3d572a9198519d1a96ee4fc02846fe6f1328348e3e74f79ead408c2c7ed4fb5824cf4a3ae94ccbeb1e7c716
Apr-05 09:48:01.057[]                 info: Validator exists in beacon chain validatorIndex=461931, pubKey=0xb8581e1c4f6280977d5cfd84341d0f429963dfe0614b3efaaf01b299b9d07ffb766e92d94f9b33ff76c841ccf10d74b1
Apr-05 09:48:01.057[]                 info: Validator statuses active=10, total=10
Apr-05 09:48:01.057[]                debug: HttpClient request routeId=getStateValidators
Apr-05 09:48:01.059[]                debug: HttpClient response routeId=getStateValidators
Apr-05 09:48:01.059[]                 info: Validator exists in beacon chain validatorIndex=461932, pubKey=0xaf0bdd50360d92fdeba5123f7a053627ebdaaeb8b176898b1ea13bed307fdcad8081e33ab171416104bebd8827bdbf93
Apr-05 09:48:01.059[]                 info: Validator exists in beacon chain validatorIndex=461933, pubKey=0xb36e5520af098a0b163d493f4fe9b0e9406ed2eb490effc87d9bc8d1c11cdb6d1aed2b7c84db053b67005a1ce969b801
Apr-05 09:48:01.059[]                 info: Validator exists in beacon chain validatorIndex=461934, pubKey=0xa33c063d4d4cf5b7ae19fceeb0ff0a22abb11feb53cb195a0b4e6f53f3863216624774f5d95f9bfcb1cbf9f03fe52d66
Apr-05 09:48:01.059[]                 info: Validator exists in beacon chain validatorIndex=461935, pubKey=0xa54358dfa9975cd884c05034c39fa51a4b8288f82d5cc063e52ac0ca3e537f316aa68eb421718f4bbef85b63c7d83b3c
Apr-05 09:48:01.059[]                 info: Validator exists in beacon chain validatorIndex=461936, pubKey=0x94d5e85442021044a72a705fdb03a768c1080e34eb977ec55b729887ce09b62abb112ce8b65de83e99d538466cf4735c
Apr-05 09:48:01.059[]                 info: Validator exists in beacon chain validatorIndex=461937, pubKey=0xa9f88897d1d0cc422517faf6c4c2211186c1807c4920d08ace23e95fa580c074b16f98851ce9d8b70e230af32bba9e9b
Apr-05 09:48:01.059[]                 info: Validator exists in beacon chain validatorIndex=461938, pubKey=0x8806312a61d339cd607517114d1c8decec82afceb85c2e10e4c7a95590aed620a5ab150119e701e23fad03b2c2d583d3
Apr-05 09:48:01.059[]                 info: Validator exists in beacon chain validatorIndex=461939, pubKey=0xb4e2a7aa20ed22adc97c8c7164b113c0a3ade2915f4d1263f1b1278a2ecbad3d7383df0ca0dffc3f685e843305d87234
Apr-05 09:48:01.059[]                 info: Validator exists in beacon chain validatorIndex=461940, pubKey=0xb99a86e1adc5baf0be2f4f688a90c259e53e3057aa4b0e931496a25f4583f13b3cb83d4a01ae6e783c8476cd8bad80e1
Apr-05 09:48:01.059[]                 info: Validator exists in beacon chain validatorIndex=461941, pubKey=0xb341016be21bee2a500d2ab70a2ceb239ff30a938996b5eb7ebba4f99f18f92939458a57e5cad5685541bd80b8999250
Apr-05 09:48:01.059[]                 info: Validator statuses active=10, total=10
Apr-05 09:48:01.060[]                debug: HttpClient request routeId=getStateValidators
Apr-05 09:48:01.134[]                debug: HttpClient response routeId=getStateValidators
```


Now here we passing in all pubkeys to `getStateValidators` which does not work if you have 500+ keys